assets/js/amp-admin-comments.js

```diff
@@ -0,0 +1,16 @@
+(function( $ ) {
+	'use strict';
+
+	/**
+	 * Move the notice below the selector and disable selector.
+	 */
+	$( document ).on( 'ready', function() {
+		var orderSelect = $( '#comment_order' ),
+			notice = $( '#amp-comment-notice' );
+		if ( orderSelect.length && notice.length ) {
+			orderSelect.prop( 'disabled', true );
+			notice.appendTo( orderSelect.parent() );
+		}
+	} );
+
+})( jQuery );
```

Comment:
@westonruter I think this could easily have been a simple inline script alongside the notice, but I made it a asset file for now, just in case.

Reply:
Thanks. I've decided to go ahead and inline it. See 310a74ade4777089816ef0136a93bc74da3b1b54

Reply:
Perfect. thought that would be the case. thanks for sorting it for me.